CIME/bless_test_results.py

```diff
@@ -216,7 +216,21 @@ def bless_test_results(
     bless_perf=False,
     **_,  # Capture all for extra
 ):
-    bless_all = not (namelists_only | hist_only | bless_tput | bless_mem | bless_perf)
+    if bless_perf:
+        bless_mem = True
+        bless_tput = True
+
+    bless_all_non_perf = not (namelists_only | hist_only | bless_tput | bless_mem)
+    perf_bless = bless_mem or bless_tput
```

Comment:
Might be worth changing `perf_bless` to `is_bless_perf`, can be confusing to have both `perf_bless` and `bless_perf`.

Reply:
Done.